.github/workflows/ci.yml

```diff
@@ -11,6 +11,9 @@ jobs:
           # - https://localhost.decentraland.services
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
```

Comment:
why 12 and not 14?

Reply:
I needed 12 for the solve challenge to work

Reply:
Fixed!